tests/foreman/ui/test_host.py

```diff
@@ -976,7 +977,7 @@ def test_positive_check_permissions_affect_create_procedure(self):
                 with self.assertRaises(UINoSuchElementError):
                     self.hosts.assign_value(
                         entity['locator'], entity['unexpected_entity'].name)
-                self.hosts.perform_action_send_escape_key()
+                self.hosts.perform_action_send_key_to_browser(Keys.ESCAPE)
```

Comment:
Let's discuss. I can't really see a value of `self.hosts.perform_action_send_key_to_browser` instead of `ActionChains(self.browser).send_keys(Keys.ESCAPE).perform()`.
Such "helper" isn't really helpful - still takes the same amount of lines and needs extra imports. 
Maybe we should think of some other implementation - let's say passing string equivalent of desired key (e.g. 'esc' instead `Keys.ESCAPE`), that way we'll at least get rid of extra import. But even with such improvement i'm not really sure it's worth separate helper to save 1 line.

Reply:
@abalakh and create a constant file of keys that has been already done by sellenium???

Reply:
That's why i'm asking for suggestions 🙂 I'm not saying we should go for it, that was one of ideas off the top of my head.
Yep, we could create a dictionary with all the keys, or, for example, take the input string and grab the webdriver's `Key` class attribute by string (getattr), this way we don't need a duplicated dict. Or we could go with something totally different, feel free to suggest.

Reply:
We might use:
`self.hosts.perform_action_send_key_to_browser("escape")`
`ActionChains(self.browser).send_keys(getattr(Keys, key.upper())).perform()`

Reply:
@oshtaier how's this idea for you?

Reply:
yeah, it is better

Reply:
can it parse, something like ctrl+shift+q?

Reply:
propose to have the two function, and the the escape one a shot cut of the more general send_keys
EG:
```python
    def perform_action_send_keys(self, **keys):
        """Perform send keys to browser"""
        ActionChains(self.browser).send_keys(**keys).perform()

    def perform_action_send_escape_key(self, wait_for_ajax=True):
        """Send escape key to browser"""
        self.perform_action_send_keys(Keys.ESCAPE)
        if wait_for_ajax:
            self.wait_for_ajax()
```

Reply:
It can't. We can use:
```
def perform_action_send_key_to_browser(keys):
    if '+' in keys:
        keys_to_send = ''.join(
            getattr(Keys, key.upper()) if key.upper() in dir(Keys) else key for
            key in keys.split('+'))
    elif keys.upper() in dir(Keys):
        keys_to_send = getattr(Keys, keys.upper())
    else:
        keys_to_send = keys
    return keys_to_send  

perform_action_send_key_to_browser('control+shift+q')
u'\ue009\ue008q'
perform_action_send_key_to_browser('escape')
u'\ue00c'
perform_action_send_key_to_browser('q')
'q'
```

to handle this.

Reply:
okay, proposed solution
```
self.hosts.perform_action_send_key_to_browser("escape")
def perform_action_send_key_to_browser
    ActionChains(self.browser).send_keys(getattr(Keys, key.upper())).perform()
    wait
```
looks good to me
```
@ldjebran we can't do like you suggested as escape is not something super special to have explicit method